testing/avara/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: shrizza <shrizza@gmail.com>
+# Maintainer: shrizza <shrizza@gmail.com>
+pkgname="avara"
+pkgver="0_git20191006"
+pkgrel=0
+pkgdesc="Port of the original 1996 Mac OS game from Ambrosia Software"
+url="https://github.com/avaraline/Avara"
+arch="all"
+license="MIT"
+depends=""
+makedepends="clang clang-dev directfb-dev glu-dev sdl2_net-dev sdl2_mixer-dev sdl2"
+options="!check"
+install=""
+subpackages="$pkgname-doc"
+source="avara https://github.com/avaraline/Avara/archive/b8700db.zip"
+builddir="$srcdir/Avara-b8700db3c696b9813b511b06a92c9faab55b282a"
+
+build() {
+	make || return 1
```

Comment:
No need for `|| return 1`, aports already do `set -e` (a failing command will fail the entire build by default)